src/lib/loaders/searchLoader.ts

```diff
@@ -7,7 +7,10 @@ export const searchLoader = gravityLoader => {
       const queryParams = {
         term: query,
         "indexes[]":
-          entities || SearchEntity.getValues().map(index => index.value),
+          entities ||
+          SearchEntity.getValues()
+            .filter(index => index.name === "gallery")
```

Comment:
We want `gallery` to be allowed as an entity, but since its more of a 'synthetic' entity and overlaps with the more proper `Profile`, it shouldn't be included by default.